okd-golang-builder.Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM quay.io/centos/centos:stream9
+
+ARG GOPATH
+
+ENV container=oci \
+    GOPATH=${GOPATH:-/go} \
+    GOMAXPROCS=8
+
+RUN mkdir -p /go/src/ && \
+    yum install -y --enablerepo=* --disablerepo=nfv-source --setopt=skip_missing_names_on_install=False \
+        bc file findutils gpgme git hostname lsof make socat tar tree util-linux wget which zip \
+        go-toolset openssl openssl-devel systemd-devel gpgme-devel libassuan-devel && \
+    yum clean all && \
+    go install github.com/josephspurrier/goversioninfo/cmd/goversioninfo@latest
```

Comment:
Lets add a note why we need this

Reply:
I don't actually know where this binary is used, but I've added comment explaining why we install it with go instead of yum